File: GuessingGameClient/GuessingGameClient/src/app/interfaces/GameDtos.ts
export interface GameDtos{
  id: number;
  secretNumber: number;
  numberOfTries: number;
  maximumTries: number;
  isFinish: boolean;
  isWinner: boolean;
  userId: number;
  gameTries:GameTries[];
}

export interface GameTries{
  guessNumber: number;
  message: string;
}

File: GuessingGameApi/GuessingGameApi/Dtos/GameDtos.cs
namespace GuessingGameApi.Dtos;

public class GameDtos
{
    public long Id { get; set; }
    public int SecretNumber { get; set; }
    public int NumberOfTries { get; set; }
    public int MaximumTries { get; set; }
    public bool IsFinish { get; set; }
    public bool IsWinner { get; set; }
    public long UserId { get; set; }
    public List<GameTriesDto>? GameTries { get; set; }
}